test/unit/jest.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
 		'<rootDir>/.*/build/',
 		'<rootDir>/.*/build-module/',
 		'<rootDir>/.+\.native\.js$',
+		'/packages/react-native-editor',
```

Comment:
Can you leave the comment that it's temporary? We should rather fix the names of the files rather and avoid this.